embed/config_logging.go

```diff
@@ -157,13 +171,12 @@ func (cfg *Config) setupLogging() error {
 
 		if !isJournal {
 			copied := logutil.AddOutputPaths(logutil.DefaultZapLoggerConfig, outputPaths, errOutputPaths)
-
-			if cfg.Debug {
-				copied.Level = zap.NewAtomicLevelAt(zap.DebugLevel)
+			copied.Level = zap.NewAtomicLevelAt(logutil.ConvertToZapLevel(cfg.LogLevel))
+			if cfg.Debug || cfg.LogLevel == "debug" {
```

Comment:
If `LogLevel` is not "debug", it should overwrite `Debug` flag here?

Reply:
`Debug` is only used here, so no need overwrite :)

Reply:
So under configuration ` etcd --debug --log-level info`, grpc tracking is enabled?

Reply:
> etcd --debug --log-level info, grpc tracking is enabled?

Yes, because `cfg.Debug` will evaluate to true.

Reply:
Sorry I think I am still confused. As an example, `etcd --debug --log-level info` sets the logging level to "info", but grpc tracking is enabled. This seems to be different than the previous behavior, where tracing is only enabled if logging level is set to "debug"?

Reply:
> As an example, etcd --debug --log-level info sets the logging level to "info", but grpc tracking is enabled. This seems to be different than the previous behavior,

I clarified with comments in the code.

Basically, we want to keep the backward compatibility since `--log-level` is the new flag in v3.4. For instance, `etcd --debug --log-level warning` in v3.4 should work in the same way as `etcd --debug` in v3.3, thus enabling the gRPC tracing even when `--log-level warning` was meant to disable gRPC tracing. If `--debug` is passed with non-debug `--log-level`, the user will be warned about its inconsistent log configuration in v3.4.

Let me know.

Reply:
Oh I misread earlier. Thanks!